NuGet.Config

```diff
@@ -28,6 +28,7 @@
     <add key="dotnet.myget.org roslyn-master-nightly" value="https://dotnet.myget.org/F/roslyn-master-nightly/api/v3/index.json" />
     <add key="dotnet.myget.org devcore" value="https://www.myget.org/F/vs-devcore/api/v3/index.json" />
     <add key="dotnet.myget.org roslyn-tools" value = "https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
+    <add key="dotnet.myget.org roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
```

Comment:
Tanner asked me to upload it to this new feed. Before it was getting pulled from roslyn-master-nightly, which is going to be deprecated soon. Correcnt @tannergooding ?